src/Mod/PartDesign/App/FeatureAddSub.cpp

```diff
@@ -64,13 +64,21 @@ short FeatureAddSub::mustExecute() const
     return PartDesign::Feature::mustExecute();
 }
 
-
-TopoShape FeatureAddSub::refineShapeIfActive(const TopoShape& oldShape) const
+TopoShape FeatureAddSub::refineShapeIfActive(const TopoShape& oldShape, const bool onlyWarn) const

```

Comment:
We're introducing an instance of boolean blindness here that I'd prefer to avoid. Can you create a more explicit enumeration instead?